src-backbone/app/js/templates/builder/pageDetails/pageElements/concepts/conceptListItemView.hbs

```diff
@@ -0,0 +1 @@
+<td class="concept">{{ name }}</td>
```

Comment:
mostly because it's tabular data — not using the table for layout, but we can always change it later